.github/ISSUE_TEMPLATE/blank.md

```diff
@@ -1,7 +1,21 @@
 ---
-name: Blank issue
-about: 'Don’t see your issue here? Open a blank issue.'
+name: 'Blank issue'
+about: "Don't see your issue here? Open a blank issue."
 title: ''
 labels: 'status: needs triage'
 assignees: ''
 ---
+
```

Comment:
The point of blank template is to be blank.

Reply:
Nah. New people need to read it to avoid low-quality/spam/unneeded issues.

Reply:
Quick search shows most of the issues are created using other templates. Not a big issue.

Reply:
I still believe that there are some new people who are going to use the wrong one. This is going to be worse since when some "new issue" links are removed, they might straight ask questions on Issues.

This needs to be mitigated nonetheless. I know a blank issues should be "blank" (even though sending it would result in a blank issue in the first place), but I think this compromise is going to be fine.